### PR TITLE
Case study for Cloud Native EdgeComputing Satellite (ENG)

### PR DESCRIPTION
**Pull request by 1Shubham7**
* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Which issue(s) this PR fixes**:

Fixes #

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

I was going through the case studies, I saw we have case studies in Mandrine for this blog but not for English, so I added one. 

* **What is the current behavior?** (You can also link to an open issue here)
We don't have an English blog for this case study

* **What is the new behavior (if this is a feature change)?**
I have added one

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

**Comment by 1Shubham7** [comment]
ptal @wbc6080 :)

File: src/pages/case-studies/satellite/index.mdx [hunk 1]
```diff
@@ -9,4 +9,73 @@ tags:
...
+This closed system architecture necessarily leads to hardware and software incompatibility, inability to interchange and reuse, 
+and very slow technological iteration.
+
+To address these issues, satellite manufacturers both domestically and internationally proposed software-defined satellites from a software-defined perspective. 
+Software-defined satellites are a new type of intelligent satellite centered on space-based computing, adopting an open system architecture that supports plug-and-play payloads and on-demand application loading. 
+They can conveniently redefine satellite functions through software updates, flexibly adapting to multiple tasks and user types. Examples include SmartSat, GSky-1, Eutelsat Quantum, and OneSat, etc.
+
+However, software-defined satellites still face numerous difficulties in software or application packaging, distribution, deployment, upgrades, and maintenance. 
+They struggle to meet the satellite's on-orbit business requirements for rapid iteration, instant deployment, grayscale release, and continuous evolution. 
+As a result, both domestic and international efforts are trying to develop a new generation of intelligent satellites.
```
Comment (wbc6080) [inline src/pages/case-studies/satellite/index.mdx:29]:
```suggestion
As a result, the industry is trying to develop a new generation of intelligent satellites.
```

File: src/pages/case-studies/satellite/index.mdx [hunk 2]
```diff
@@ -9,4 +9,73 @@ tags:
...
+By utilizing cloud-native edge computing technology, satellites can process data in space, significantly reducing transmission pressure and meeting the diverse needs of aerospace fields. 
+
+**Significance: Giving Satellites "Apps", Enabling "Business in Space and Services in Orbit"**
+
+As shown in the diagram, by reconstructing the KubeEdge system and lightweight AI inference model Sedna according to satellite characteristics, empowering satellites with cloud-native edge computing capabilities, and achieving collaborative AI inference between satellites and ground stations, the following was accomplished:
+- In remote sensing scenarios, on-orbit target identification accuracy improved by over 50%
+- Satellite data transmission volume reduced by 90%
+- Support for in-orbit continuous AI model updates
+
+![Satellite](images/satellite.png)
```
Comment (wbc6080) [inline src/pages/case-studies/satellite/index.mdx:58]: Do you have an editable version of this image? Maybe we can also replace the Chinese in the picture with English

Comment (1Shubham7) [reply src/pages/case-studies/satellite/index.mdx:58]: I had this english one so added : ) also Elias, a lot of Indian devs were interested in these case studies so I will create english versions for other case studies as well, please review them as well : )

Comment (wbc6080) [reply src/pages/case-studies/satellite/index.mdx:58]: Sounds great! Just go ahead!

**Review by wbc6080 — COMMENTED** [review]
/lgtm

**Review by fisherxu — COMMENTED** [review]
/approve

Thanks @1Shubham7

**Pull request merged**